mappings/com/mojang/realmsclient/RealmsAvailability.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_aaohlqkp com/mojang/realmsclient/RealmsAvailability
```

Comment:
`net/minecraft/realms` please! we should probably look into matching the mojang packages here, but we're not going to throw one class in a brand new place for this pr